receiver/mysqlreceiver/integration_test.go

```diff
@@ -60,15 +59,15 @@ func TestMySqlIntegration(t *testing.T) {
 		require.Eventuallyf(t, func() bool {
 			return len(consumer.AllMetrics()) > 0
 		}, 2*time.Minute, 1*time.Second, "failed to receive more than 0 metrics")
-
-		md := consumer.AllMetrics()[0]
-		require.Equal(t, 1, md.ResourceMetrics().Len())
-		ilms := md.ResourceMetrics().At(0).InstrumentationLibraryMetrics()
-		require.Equal(t, 1, ilms.Len())
-		metrics := ilms.At(0).Metrics()
 		require.NoError(t, rcvr.Shutdown(context.Background()))
 
-		require.Equal(t, len(metadata.M.Names()), metrics.Len())
+		actualMetrics := consumer.AllMetrics()[0]
+
+		expectedFile := filepath.Join("testdata", "integration", "expected.5_7.json")
```

Comment:
Maybe I'm missing something here, but should expected be the 8.0 version here?

Reply:
Yup, crossed wires. Good catch - I'll fix.